### DISCUSSION TITLE
Comparte two JSBSim properties in a script

### DISCUSSION THREAD

**Discussion by LucasJSch** [JSBSim-Team/jsbsim#1199] [Q&A]
I want to do something like: `position/h-sl-ft gt customized/custom-target ` in an XML script.

Is it possible?

**Comment by bcoconni** [comment]
Sorry for the late answer ! Yes this syntax is legal and is used in the example script `scripts/J2461.xml`:
https://github.com/JSBSim-Team/jsbsim/blob/2bdf0d40ad2c39be0fa581f93cf1569883ef6077/scripts/J2461.xml#L97

---

For the record, according to the code in `FGCondition`, the left item in the comparison must be a property while the right item can either be a property or a value:
https://github.com/JSBSim-Team/jsbsim/blob/2bdf0d40ad2c39be0fa581f93cf1569883ef6077/src/math/FGCondition.h#L80-L81
https://github.com/JSBSim-Team/jsbsim/blob/2bdf0d40ad2c39be0fa581f93cf1569883ef6077/src/math/FGCondition.cpp#L192-L204
and both `FGPropertyValue` and `FGRealValue` inherit from `FGParameter`:
https://github.com/JSBSim-Team/jsbsim/blob/2bdf0d40ad2c39be0fa581f93cf1569883ef6077/src/math/FGPropertyValue.h#L62
https://github.com/JSBSim-Team/jsbsim/blob/2bdf0d40ad2c39be0fa581f93cf1569883ef6077/src/math/FGRealValue.h#L57
